Lib/test/libregrtest/utils.py

```diff
@@ -535,6 +548,30 @@ def is_cross_compiled():
     return ('_PYTHON_HOST_PLATFORM' in os.environ)
 
 
+def format_resources(use_resources: tuple[str, ...]):
+    use_resources = set(use_resources)
+    all_resources = set(ALL_RESOURCES)
+
+    # Express resources relative to "all"
+    relative_all = ['all']
+    for name in sorted(all_resources - use_resources):
+        relative_all.append(f'-{name}')
+    for name in sorted(use_resources - all_resources):
+        relative_all.append(f'{name}')
```

Comment:
```suggestion
        relative_all.append(name)
```

Reply:
Oh right. Sadly, I saw your comment after the change was merged. I make try to include it in my next regrtest change ;-)